kinder/ci/tools/update-workflows/pkg/workflows.go

```diff
@@ -105,9 +106,15 @@ func processWorkflows(settings *Settings, cfg *jobGroup, oldestVer, minVer *vers
 		}
 
 		// add header and write output file
+		str = autogeneratedHeader + "\n" + str
```

Comment:
both the testinfra and workflow files  are validated with this "AUTOGENERATED ..." header present for consistency.